tests/end_to_end/conftest.py

```diff
@@ -243,7 +243,7 @@ def code_path(request):
 @pytest.fixture
 def subproject_id(request):
     mark = request.node.get_closest_marker("subproject_id")
-    return mark.args[0] if mark else None
```

Comment:
maybe it's the other way around: this fixture should return None, but where we use it, we should do `json.dumps(subproject_id)` (either are fine)

Reply:
Sure, I was just trying to get the tests working